src/style.css

```diff
@@ -0,0 +1,112 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-bold";
+  font-style: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+form {
+  margin: 20px 0;
+}
+
+.searh-bar-big {
+  font-family: Avenir;
+  width: 100%;
+  height: 70px;
+  font-weight: 300px;
```

Comment:
font-weight can't be 300px
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight

Reply:
fixed

Reply:
code still have font-weight: 300px;
![image](https://user-images.githubusercontent.com/70009205/232832396-9340875e-1fd4-4838-8b75-3b8e1185fba4.png)